Bayesian_Cognitive_Modeling/ParameterEstimation/DataAnalysis/Kappa.stan

```diff
@@ -0,0 +1,55 @@
+// #### Notes to Stan model #######################################################
+// ## 1) This is the first time we use simplex data type. Simplex is similar to 
+// ##    vector, but with a property that the sum of all its elements is equal to 1.
+// ## 2) Sampling statements for parameters alpha, beta and gamma could be removed
+// ##    leading to uniform prior on (0, 1) interval which is the same as beta(1, 1)
+// ## 3) Variable n was removed here. Stan doesn't need this information as
+// ##    an argument for multinomial distribution. Always make sure that you know 
+// ##    what arguments are required for a function / sampling statement. In many 
+// ##    cases these are different from BUGS. Very useful for this are last pages 
+// ##    of Stan manual
+// ################################################################################
+
+// Kappa Coefficient of Agreement
```

Comment:
Cohen's `kappa` is a pretty bad way of measuring agreement.  You're much better off with a model of annotation.  I've written a couple papers about this as I used to be in NLP and crowdsourcing and evaluating agreement is what dragged me into Bayesian stats.  Not for this PR, though :-).